django/conf/locale/__init__.py

```diff
@@ -73,6 +73,12 @@
         "code": "ca",
         "name": "Catalan",
         "name_local": "català",
+    },
+     "ckb": {
+        "bidi": False,
```

Comment:
Is Sorani not a RTL language?
```suggestion
        "bidi": True,
```
We should also add it to the `LANGUAGES_BIDI` in the `global_settings.py`

Reply:
Yes, Kurdish Sorani is an RTL language. Thanks.